<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>prodialog</class>
 <widget class="QWidget" name="prodialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1072</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="img">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-10</y>
     <width>1071</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">image: url(:/dashboard/dashboard.png);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="editpro">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>110</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(38, 0, 115);
background-color: rgb(216, 213, 184);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string> Edit Project</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="idLineEdit">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>290</y>
     <width>121</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(181, 181, 181);</string>
   </property>
  </widget>
  <widget class="QPushButton" name="deletepro">
   <property name="geometry">
    <rect>
     <x>770</x>
     <y>150</y>
     <width>191</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(38, 0, 115);
background-color: rgb(216, 213, 184);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>Delete Project</string>
   </property>
  </widget>
  <widget class="QComboBox" name="roleComboBox">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>290</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: rgb(140, 140, 140);</string>
   </property>
   <item>
    <property name="text">
     <string notr="true">Role</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Admin</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>User</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="addOrEditUser">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>340</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 238, 205);
background-color: rgb(113, 113, 113);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>Add or Edit User</string>
   </property>
  </widget>
  <widget class="QPushButton" name="delete_user">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>390</y>
     <width>181</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(241, 238, 205);
background-color: rgb(113, 113, 113);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>Delete User</string>
   </property>
  </widget>
  <widget class="QLabel" name="idLabel">
   <property name="geometry">
    <rect>
     <x>370</x>
     <y>250</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Members  :</string>
   </property>
  </widget>
  <widget class="QLabel" name="nameLabel">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>40</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 157);
background-color: rgb(241, 238, 205);
border-radius:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="descriptionLabel">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>90</y>
     <width>271</width>
     <height>101</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 157);
background-color: rgb(241, 238, 205);
border-radius:7px;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="combo">
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>270</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:6px;
background-color: rgb(197, 197, 197);</string>
   </property>
   <item>
    <property name="text">
     <string>Active</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Inactive</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>760</x>
     <y>270</y>
     <width>63</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Status :</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>810</x>
     <y>320</y>
     <width>93</width>
     <height>29</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(38, 0, 115);
background-color: rgb(216, 213, 184);
border-radius:6px;</string>
   </property>
   <property name="text">
    <string>Change</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>140</y>
     <width>141</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 10pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>Current Situation :</string>
   </property>
  </widget>
  <widget class="QLabel" name="currentstatus">
   <property name="geometry">
    <rect>
     <x>120</x>
     <y>190</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(0, 0, 157);
text-align: center;
background-color: rgb(241, 238, 205);
border-radius:7px;
</string>
   </property>
   <property name="text">
    <string>ÙŽActive</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
